spec/models/inventory_items_spec.rb

```diff
@@ -1,58 +1,29 @@
 require 'rails_helper'
 
-RSpec.describe InventoryItemAdapter, :type => :model do
-  let(:inventory_item_adapter) { InventoryItemAdapter }
-
-  describe '#connected_with_octopus?' do
-    subject { inventory_item_adapter.connected_with_octopus? }
-
-    it { is_expected.to be_truthy }
-  end
-
-  describe '#connection' do
-    subject { inventory_item_adapter.connection.current_shard }
-
-    it 'current_shard is equal to grossman' do
-      expect(is_expected.to eq(:grossman))
+RSpec.describe InventoryItem, :type => :model do
+  let(:inventory_item) do
+    Octopus.using(:grossman) do
+      InventoryItem.new
     end
   end
-  describe '#all' do
-    before do
-      Octopus.using(:grossman) do
-        create(:inventory_item)
-      end
-    end
 
-    subject(:inventory_items) { inventory_item_adapter.all }
+  subject { inventory_item.attributes.to_snake_keys }
+  it { is_expected.to be_truthy }
 
-    it 'empty is not false' do
-      expect(inventory_items.empty?).to be_falsey
-    end
-    describe 'first item' do
-      subject { inventory_items.first }
-
-      it { is_expected.to have_key(:item_id) }
-      it { is_expected.to have_key(:in_item_description) }
-    end
-  end
-  describe '#like_id_description' do
-    before do
+  it { is_expected.to have_key(:item_id) }
+  it { is_expected.to have_key(:in_item_description) }
+  it { is_expected.to have_key(:uuid) }
+  describe '#init' do
```

Comment:
I think you should actually create a test for new, and then skip using factory girl since you don't actually need to persist to the database to accomplish the tests below.